src/components/ping.js

```diff
@@ -0,0 +1,26 @@
+import Konva from 'konva'
+
+export const addPing = (e, layer) => {
+  const amplitude = 25
+  const period = 500
+
+  let item = new Konva.Circle({
```

Comment:
You can declare item as a const since it's not reassigned later

Reply:
For sure should be using consts always when possible and later change them to let, if required.

Reply:
yeah no problem. Will fix